crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -61,15 +65,30 @@ fn initialize<B: Board>(mut call: SchedulerCall<B, api::initialize::Sig>) {
     let api::initialize::Params { algorithm } = call.read();
     let scheduler = call.scheduler();
     let result = try {
-        let context = match convert_hash_algorithm::<B>(*algorithm)?? {
+        let context_or_error = match convert_hash_algorithm::<B>(*algorithm)?? {
```

Comment:
nit: it's ok to call a `Result<T, Error>` the same as a `T`, otherwise the codebase will be full of `or_error`
```suggestion
        let context = match convert_hash_algorithm::<B>(*algorithm)?? {
```